### ISSUE TITLE
Calamares is not accessible by screen reader (orca) .

### ISSUE THREAD

**Issue by calamares-user** [calamares/calamares#470]
Jira issue originally created by user libregeekingkid:

Screen reader Orca can not read the text in the Calamares Installer. Orca seems to read the text with Ubiquity fine but is failing to do so with Calamares. 
As per my discussion with an accessibility expert, It is a problem with most of the softwares having Qt components. So right now Ubiquity is the only Installer supporting accessibility standards. 

Steps to reproduce:
1) Boot a live ISO of Tanglu Chromodoris
2) Select Try Tanglu Live
3) once the system is fully booted start the screen reader from the accessibility settings or using the keyboard shortcut Super <ins> Alt </ins> S
4) You will notice that Orca will start reading key presses or Window frame titles.
5) now start the Installl Tanglu window. 
6) Orca will not be able to read anything on the Installer window, not even the key presses.

**Comment by adriaandegroot** [comment]
Longer discussion and explanation -- and link to Qt fix which is the "quick" fix for this issue -- in #523

**Comment by zersiax** [comment]
Sorry to necromance this issue but where are we actually at with this, currently? The linked issue doesn't seem very hopeful and it has seemingly been quiet around this since 2020. Can Orca work with Calamares as of today?
I'm helping out a developer who wants to do the right thing but their OS uses Calamares Installer. DO they need to do anything specific to make sure this will work?

**Comment by demmm** [comment]
For future comments, please continue at https://github.com/calamares/calamares/issues/1294
Recent comments there are giving pointers for a possible approach.
